grafana/risingwave-dashboard.py

```diff
@@ -827,7 +827,14 @@ def section_hummock(panels):
                 "sum(rate(state_bloom_filter_check_counts[$__rate_interval])) by (job,instance)", "bloom filter check count  - {{job}} @ {{instance}}"
             ),
         ]),
-        
+        panels.timeseries_ops("Iter keys flow", [
+            panels.target(
+                "sum(rate(state_bloom_filter_check_counts[$__rate_interval])) by (instance)", "iter total key count - @ {{instance}}"
```

Comment:
this two panels use same promql ?

I think it will be `processed_key_count` and `scan_key_count` ?